### PR TITLE
Handle unspecified background color of default face

### PR DESCRIPTION
**Pull request by doolio**
(face-background 'default) is unspecified in "no-windows" mode.This incorporates juergenhoetzel/profile-dotemacs@ff4e41cea0afd70f80d531267e492a8803d64ac3 into your repository. As with my first PR if you accept this PR please correct any errors as I'm only learning to programme.

### CODE REVIEW

**Comment by raxod502** [comment]
This needs to be rebased now that #2 was merged.

**Comment by doolio** [comment]
My first rebase \O/ hopefully I did it right.

File: profile-dotemacs.el [hunk 1]
```diff
@@ -140,10 +140,18 @@ grayed out.")
...
                percentage
                (face-background 'default)
                (face-background 'profile-dotemacs-time-face)))
+=======
+	      (profile-dotemacs-percentage-color
+	       percentage
+               (if (color-defined-p (face-background 'default))
+                   (face-background 'default) "black")
+	       (face-background 'profile-dotemacs-time-face)))
+>>>>>>> Handle unspecified background color of default face
```
Comment (raxod502) [inline profile-dotemacs.el]: You have committed merge markers. See https://stackoverflow.com/q/10657315/3538165.

**Review by raxod502 — CHANGES_REQUESTED** [review]

**Comment by doolio** [comment]
I've removed those conflict markers with a new commit. Was that the correct action or should one squash the commits into one? Thanks for your patience.

**Review by raxod502 — APPROVED** [review]

**Comment by raxod502** [comment]
It would probably make more sense to squash the commits into one. However, GitHub lets me do that while merging the pull request, so I did.

**Comment by doolio** [comment]
Great, thanks.

**Pull request merged**